packages/components/src/components/progress-bar/progress-bar.tsx

```diff
@@ -0,0 +1,51 @@
+import { Component, Prop, h } from '@stencil/core';
+import { CssClassMap } from '../../utils/utils';
+import classNames from 'classnames';
+
+@Component({
+  tag: 't-progress-bar',
+  styleUrl: 'progress-bar.css',
+  shadow: true,
+})
+export class ProgressBar {
+  /** (required) progress percentage */
+  @Prop() public percentage: number;
+  /** (optional) show progress variant */
+  @Prop() public variant?: string;
+  /** (optional) progress stroke width */
+  @Prop() public strokeWidth?: number;
+  /** (optional) show progress percentage text */
+  @Prop() public showText?: boolean;
+  /** (optional) progress text display inside bar */
+  @Prop() public textInside?: boolean;
+
+  public render() {
+    return (
+      <div class="progress-bar">
+        <div
+          class="progress-bar__outer"
+          style={{ height: this.strokeWidth ? `${this.strokeWidth}px` : '6px' }}
```

Comment:
better to assign a default value to the prop, then you don't need to have a condition here

`@Prop() public strokeWidth?: number = 6;`

`style={{ height: this.strokeWidth }}`

Reply:
updated